region/tourian/main/Mother Brain Room.json

```diff
@@ -138,7 +182,18 @@
               "name": "Base",
               "notable": false,
               "requires": [
-                "h_canOpenZebetites"
+                "h_canOpenZebetites",
+                {"or": [
+                  {"and": [
+                    "canDodgeWhileShooting",
```

Comment:
Dodging actually increases the ammo cost since Zebetites regenerate.  With Supers the pattern needs to be pretty bad to need another Super, so 1/3.  But Missiles are going to need more missiles if you stop shooting at all.

Turrets have boxy hitboxes, so like, trying to shoot zebs from range makes it more likely you get blocked.  If health doesn't matter, you point blank the Zebs.

I was doing a lot of `canResetFallSpeed` to dodge things, didn't do much rinka killing since that makes menuing harder and they respawn instantly.  It was still really hard to clear 4 zebs + MB1 taking only 2 turret hits.  Missile only was hard to do suitless no etanks.  Rinka hits or falling in acid was deadly.

Zeb1 followed by Glass case were the easiest to no-ice dodge.